config/local.json

```diff
@@ -50,5 +50,12 @@
     "twitter": {
         "callback": "http://local.delphishq.com:8080/twitter/callback",
         "redirect": "http://local.delphishq.com:8000"
+    },
+    "s3_bucket": {
```

Comment:
Hmm, for true local we should consider makign this point to a local directory instead of actually requiring S3.